packages/terra-clinical-item-collection/src/ItemCollection.jsx

```diff
@@ -2,123 +2,133 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import ResponsiveElement from 'terra-responsive-element';
 import 'terra-base/lib/baseStyles';
+
+import Item from './Item';
+import ListView from './_ListView';
+import TableView from './_TableView';
 import './ItemCollection.scss';
 
-import createListView from './_CreateListView';
-import createTableView from './_CreateTableView';
+const maxDisplays = 8;
 
 const propTypes = {
   /**
-  * The breakpoint to switch from a table view to a list view. Breakpoint options are tiny, small, medium, large, or huge.
-  */
+   * The breakpoint to switch from a table view to a list view. Breakpoint options are tiny, small, medium, large, or huge.
+   */
   breakpoint: PropTypes.oneOf(['tiny', 'small', 'medium', 'large', 'huge']),
   /**
-   * The styles to spread to the table. Table style options are isPadded and isStriped.
+   * The items that will flex between either a list item or a table row.
+   */
+  children: PropTypes.node.isRequired,
+  /**
+  * Whether or not the item collection should create a layout to display a start accessory.
+  */
+  hasStartAccessory: PropTypes.bool,
+  /**
+  * The number of displays the item collection should create a layout for.
+  */
+  numberOfDisplays: PropTypes.number,
+  /**
+  * Whether or not the item collection should create a layout to display a comment.
```

Comment:
I feel these documentation comments are too implementation specific, and a bit confusing to the consumers. Instead of saying create a layout, I think it's just fine to say 'where or not the item collection contains a comment'.